data/json/furniture_and_terrain/terrain-walls.json

```diff
@@ -7,7 +7,10 @@
       { "item": "rock", "count": [ 0, 2 ] },
       { "item": "wood_panel", "count": [ 0, 1 ] },
       { "item": "nail", "charges": [ 2, 8 ] },
-      { "item": "splinter", "count": [ 1, 5 ] }
+      { "item": "splinter", "count": [ 1, 5 ] },
+      { "item": "pipe", "count": [ 0, 2 ] },
+      { "item": "cu_pipe", "count": [ 0, 2 ] },
+      { "item": "wire", "count": [ 1, 5 ] }
```

Comment:
Bashing EVERY wall will yield at least one stack of wire?

Reply:
Not anymore.

Reply:
> will yield at least one stack of wire?

This is the type of wire you find in fences, not copper wire - there aren't stacks of this type of wire, count 1 will give you one wire item.